qna.md

```diff
@@ -0,0 +1,77 @@
+
+Q: Esiste un code of conduct?
+A: Il code of conduct è disponibile [qui](https://github.com/Il-Libro-Open-Source/book/blob/main/CODE_OF_CONDUCT.md)
+
+Q: Quale è la governance?
+A: Il libro viene mantenuto open e fruibile da tutti con una governance che vuole essere aperta, per questo da subito sono stati nominati una serie di figure dette Ambassador elette con voto su GH per gestire la crescita del libro, le posizioni vengono aperte temporalmente, questo periodo è in fase decisionale.
+
+Q: Quali figure sono presenti?
+A: Esistono due figure, l'Ambassador e il Contributor.
+
+Q: Chi ne fa parte?
+A: L'elenco è aggiornato qui: https://github.com/Il-Libro-Open-Source/governance/blob/main/AMBASSADOR.md
+
+Q: In che maniera viene destituito un ambassador?
+A: Un ambasssador può esser destituito con questi criteri: [https://github.com/Il-Libro-Open-Source/governance/blob/main/guidelines/ambassador-allontanamento.md](https://github.com/Il-Libro-Open-Source/governance/blob/main/guidelines/ambassador-allontanamento.md)
+
+Q: Che ruolo svolge il contributor?
+A: Il contributor è una figura che collabora attivamente come un Ambassador ma non è referente per un particolare capitolo.
+
+Q: Chi ne fa parte?
+A: L'elenco è fruibile [qui](https://github.com/Il-Libro-Open-Source/governance/blob/main/CONTRIBUTOR.md)
+
+Q: In che maniera viene eletto un contributor?
+A: Il regolamento per il contributor è il medesimo degli ambassador.
+
+Q: In che maniera viene destituito un contributor?
+A: Il regolamento per il contributor è il medesimo degli ambassador.
+
+Q: Cosa sono i working group? e chi ne fa parte?
+A: Sono gruppi di lavoro specializzati in vari aspetti come es. la revisine dei contenuti, trovi maggiori infomazioni [qui ](https://github.com/Il-Libro-Open-Source/governance/blob/main/WG.md)
+
+Q: Come posso partecipare a uno di questi working group?
+A: Ogni gruppo può decidere di allargare in caso di necessità il gruppo e apre le candidature per gli ambassador che sono interessanti, successivamente sarà creato un sondaggio.
+
+Q: Mi sono accorto che alcuni contenuti non sono in linea col COC, come posso segnalarli?
+A: Nel COC è presente un link al form con il quale contattare direttamente il workgroup che analizzerà il caso.
+
+Q: Quali sono le linee guida per scrivere contenuti?
+A: Potete trovare un'introduzione [qui](https://github.com/Il-Libro-Open-Source/book/blob/main/GUIDELINES.md)
+
+Q: Come posso contribuire?
+A: La contribuzione è aperta che si sia contributor, ambdassador o nessuno dei due, si può seguire la linea guida [qui](https://github.com/Il-Libro-Open-Source/book/blob/main/GUIDELINES-CONTENUTI.md)
+
+Q: Come si contribuisce? Con che mezzi?
+A: Attualmente si contribuisce tramite git lavorando sul repository principale come fosse codice.
```

Comment:
Questa la vedo come domanda inglobata in quella sopra una volta che linki al CONTRIBUTING

Reply:
Fatto.